userguide/kaui/includes/overdue.adoc

```diff
@@ -0,0 +1,188 @@
+:icons: font
+:imagesdir: C:\_My Documents\FlowWritingLLC\Projects\Kill Bill\Documentation\killbill-docs\userguide\assets\img\kaui
+
+=== Overview
+
+Kaui lets you configure basic settings for how the system behaves when an account is overdue. In Kaui, this is a subset of the full overdue configuration settings and is defined at the tenant level. For more information, see the https://docs.killbill.io/latest/overdue.html[_Overdue System Guide_].
+
+[NOTE]
+*Note:* You can also configure overdue actions for the _global_ Kill Bill system (as opposed to configuring at the tenant level). For more information, see the https://docs.killbill.io/latest/overdue.html[_Overdue System_] documentation.
+
+See the following sections to configure overdue (dunning) flows for a tenant:
+
+[cols="1,1"]
+[frame=none]
+[grid=none]
+|===
+
+a| * <<View Overdue Configuration,View overdue configuration>>
+* <<Configure Overdue Flows,Configure overdue flows>>
+* <<Upload an XML Overdue Configuration File,Upload an XML overdue configuration file>>
+
+a| * <<View Overdue Configuration as XML,View overdue configuration as XML>>
+* <<Delete Overdue Flows,Delete overdue flows>>
+
+|===
+
+You cannot edit a raw overdue configuration XML file in Kaui.
+
```

Comment:
Would it be better to move this line to the "Upload an XML Overdue Configuration File" section?

Reply:
Fixed - see page 4 of PDF